Dockerfile

```diff
@@ -72,12 +72,17 @@ USER nobody
 # LAYER: Keygen
 # An alternative build target that executes the keygeneration
 # =================================================================================================
-# hadolint ignore=DL3029
-FROM --platform=linux/amd64 eclipse-temurin:17-jre-alpine AS keygen
+FROM eclipse-temurin:17.0.7_7-jre AS keygen
```

Comment:
As you can see [here](https://hub.docker.com/_/eclipse-temurin/tags?page=1&name=alpine) alpine does not have arm releases. Switching to the base release of this.